Update sdk release notes to reflect 1.0.1 changes (#4430)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/sdk/1.0.x/release-notes/index.md

menuWeight: 0
---

Kaptain SDK 1.0.1 Release Notes

### New Features

> ### Improvements and Fixes

Reviewer: hi, I think we can mention here an SDK only parts:

```
* Fix incorrect handling of the S3 endpoint string
* New `timeout` parameter for `Model.train` and `Model.tune` methods 
* Upgrade from KFServing 0.6.1 to Kserve 0.7.0
* Upgrade kubeflow-training to 1.4.0
```

* Fix incorrect handling of the S3 endpoint string
* Remove `FailedScheduling` from the events list, introduce timeout for train and tune methods
* Migrate to KServe v0.7.0
* Use default minio tenant image
* Upgrade training operator to 1.4.0

> * Don't mutate constants in _check_label_exists

Author: ```suggestion
* Do not mutate constants in _check_label_exists
```

> * Fix incorrect handling of the S3 endpoint string
> * Remove `FailedScheduling` from the events list, introduce timeout for train and tune methods
> * Migrate to KServe v0.7.0
> * Use default minio tenant image
> * Upgrade training operator to 1.4.0
> * Do not mutate constants in _check_label_exists
> * Handle missing / problem values more gracefully in utilities methods

Author: ```suggestion
* Fix incorrect handling of the S3 endpoint string
* New `timeout` parameter for `Model.train` and `Model.tune` methods
* Upgrade from KFServing 0.6.1 to Kserve 0.7.0
* Upgrade kubeflow-training to 1.4.0
```

### Breaking changes

> Given the migration from KFServing toi KServe this SDK version is only compatible with Kaptain 2.0

Author: ```suggestion
Given the migration from KFServing to KServe this SDK version is only compatible with Kaptain 2.0
```

menuWeight: 0
---

Kaptain SDK 1.0.1 Release Notes

### New Features

> No new features for this release.

Reviewer 2: @juliangieseke added this just to have something underneath the header and indirectly provide the info that this release is only about improvement and fixes.

Author: I think Id rather remove the header entirely (missed that). Its more common not to have the header rather than stating no changes.

menuWeight: 0
---

Kaptain SDK 1.0.1 Release Notes

### New Features

### Improvements and Fixes
* Fix incorrect handling of the S3 endpoint string
* Remove `FailedScheduling` from the events list, introduce timeout for train and tune methods
* Migrate to KServe v0.7.0
* Use default minio tenant image
* Upgrade training operator to 1.4.0
* Don't mutate constants in _check_label_exists
* Handle missing / problem values more gracefully in utilities methods

### Breaking changes
Given the migration from KFServing toi KServe this SDK version is only compatible with Kaptain 2.0

> 
> [quick-start]: ../../../tutorials/sdk/quick-start
> [image-builder]: ../../../tutorials/sdk/image-builder

Author: ```suggestion

```
